src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -16,7 +16,7 @@
 /**
  * Used to handle the data validation aspect e.g. validate emails, names, etc.
  */
-public class FieldValidator {
+public final class FieldValidator {
```

Comment:
`FieldValidator` still has a public constructor. So, other classes can still create instances of `FieldValidator`. We want to prevent this since we are changing it to a utility class.

Making this change will require you to update methods in other classes that call methods from `FieldValidator` as well.

Reply:
Changes have made successfully. :)